scripts/build-changelog.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+/*
+* This script generates changelog based on the json files in "../.changes/nextrelease/"
+* and generates a new TAG for the build.
+*/
+
+$dir = "../.changes/";
+$newServiceFlag = False;
+
+
+function readChangelog($dir){
+	$dir = $GLOBALS['dir']."/nextrelease/";
+	$CHANGELOG=[];
+	if (is_dir($dir)){
```

Comment:
nextrelease directory will be inside a " .changes" folder, it will have release notes for next release. I just added checks to make sure that the directory is there and it exists.  even for a doc only update the release notes will be inside the nextrelease folder.